plugins/org.polarsys.capella.filtering.model.edit/plugin.xml

```diff
@@ -9,7 +9,7 @@
 <!--@generated-->
   <extension point="org.eclipse.emf.edit.itemProviderAdapterFactories">
       <!-- @generated filtering -->
-      <factory class="org.polarsys.capella.filtering.provider.FilteringItemProviderAdapterFactory" supportedTypes="org.eclipse.emf.edit.provider.IEditingDomainItemProvider                org.eclipse.emf.edit.provider.IStructuredItemContentProvider                org.eclipse.emf.edit.provider.ITreeItemContentProvider                org.eclipse.emf.edit.provider.IItemLabelProvider                org.eclipse.emf.edit.provider.IItemPropertySource" uri="http://www.polarsys.org/capella/filtering/1.4.0"/>
+      <factory class="org.polarsys.capella.filtering.provider.FilteringItemProviderAdapterFactory" supportedTypes="org.eclipse.emf.edit.provider.IEditingDomainItemProvider                org.eclipse.emf.edit.provider.IStructuredItemContentProvider                org.eclipse.emf.edit.provider.ITreeItemContentProvider                org.eclipse.emf.edit.provider.IItemLabelProvider                org.eclipse.emf.edit.provider.IItemPropertySource" uri="http://www.polarsys.org/capella/filtering/1.4.1"/>

```

Comment:
Changing metamodel imply migration. Are you sure of that for a patch release?

Reply:
Can we bump the version of the addon and keep the metamodel version as it is ? I think it's better to skip migration issues !